Jenkinsfile

```diff
@@ -34,12 +34,22 @@ pipeline {
                                 try {
                                     checkout scm
                                     sh 'make -f docker.Makefile cross e2e-cross tars'
+                                    sh 'BIN_NAME=yamlschema make -f docker.Makefile cross'
                                     dir('bin') {
                                         stash name: 'binaries'
                                     }
                                     dir('e2e') {
                                         stash name: 'e2e'
                                     }
+                                    dir('vendor') {
```

Comment:
we need `vendor` for the json schema right ? :thinking:

Reply:
Yep... 😞

Reply:
Stashing involves sending files from the worker to the master and unstashing the same files from the master to the worker…

Reply:
I know... but the other solution I see would be to `checkout scm`, and fall back again to the windows carriage return problem... unless I override the e2e directory by unstashing it after the checkout :trollface: